pkg/storage/internal/integrationtests/internalize_create_process_test.go

```diff
@@ -22,6 +22,7 @@ func TestInternalizeThenCreateThenProcess(t *testing.T) {
 	given := testabilities.Given(t)
 	activeStorage := given.Provider().
 		WithRandomizer(randomizer.NewTestRandomizer()).
+		WithServices().
```

Comment:
Couldn't it be a default? And only if somewhere we would really need it without services, then we could call `WithoutServices`